src/styles.scss

```diff
@@ -203,7 +203,7 @@ body {
     width: $stars-width;
 
     mat-icon {
-      margin: 0 $stars-margin;
+      padding: 0 1px;

```

Comment:
@robinGiri Use the scss variable `$stars-margin`.  This will help to maintain this if we make changes to the layout